ksp_plugin/part.cpp

```diff
@@ -174,13 +179,36 @@ not_null<std::unique_ptr<Part>> Part::ReadFromMessage(
     serialization::Part const& message,
     std::function<void()> deletion_callback) {
   bool const is_pre_cesàro = message.has_tail_is_authoritative();
-  not_null<std::unique_ptr<Part>> part =
-      make_not_null_unique<Part>(message.part_id(),
-                                 message.name(),
-                                 Mass::ReadFromMessage(message.mass()),
-                                 DegreesOfFreedom<Barycentric>::ReadFromMessage(
-                                     message.degrees_of_freedom()),
-                                 std::move(deletion_callback));
+  bool const is_pre_fréchet = message.has_mass();
+
+  std::unique_ptr<Part> part;
+  if (is_pre_fréchet) {
+    static constexpr MomentOfInertia zero;
+    static constexpr MomentOfInertia one = SIUnit<MomentOfInertia>();
+    InertiaTensor<Barycentric> inertia_tensor(
+        Mass::ReadFromMessage(message.mass()),
+        R3x3Matrix<MomentOfInertia>({one, zero, zero},
+                                    {zero, one, zero},
+                                    {zero, zero, one}),
```

Comment:
As discussed, construct a reasonable moment of inertia; I think here you have a ball of radius sqrt(5 kg m<sup>2</sup> / (2m)), i.e., a ball that shrinks as it gets heavier!

Reply:
Done.